_posts/2018-05-15-jiractl.md

```diff
@@ -0,0 +1,105 @@
+2	  kanban    Sharks
+```
+
+_Intuitive syntax_ -- Our developers use kubernetes and kubectl, which allows users to set a context and then perform actions within that context; we wanted to emulate that syntax in our tool since it’s familiar to developers, making adoption easier, and more importantly, since it reflects how we use Jira: performing a set of actions (e.g. viewing past sprint velocities) within a context (e.g. Jira board). Like kubectl, jiractl uses the `get` action to give an overview of a resource, and `describe` to provide detailed insight.
+
+For example, you can get a team’s sprints by running:
+```console
+$ jiractl --team=1 get sprints
+```
+
+And fetch details with:
+```console
+$ jiractl --team=1 describe sprints
+```
+
+_Multiple views_ -- Many of the existing tools we saw were crafted around an issue-based workflow, but we wanted one that would let us run an entire sprint planning or backlog refinement session from the command line. Others had more of the features we were looking for but were still in the early stages of development. To fulfill our use cases, we wanted jiractl users to be able to answer questions like “What was my team’s velocity over the past few sprints?” and “What epics do we have ready?” We found those were easiest to answer by configuring different commands for each of those views.
+
+To view a team’s epics you can run:
+```console
+$ jiractl --team=1 get epics
+Epic:
+KEY		SUMMARY
+GX-4019 	Build Experimentation Platform
+GX-5212 	Self-service A/B results dashaboard
+GX-5589 	Add NPS Scoring
+```
+
+To get a particular sprint:
+```console
+$ jiractl --team=1 get sprint { sprint_id }
+NAME               STARTDATE				ENDDATE				COMPLETED/TOTAL POINTS
+Sprint 5/7-5/18    2018-05-07T09:33:33.302-07:00	2018-05-18T17:40:00.000-07:00	8/13
+
+Members: Gabriel Isenberg, Jon Yonker
+
+Issues:
+KEY		STATUS		SUMMARY			     POINTS
+GX-1199		In Progress     Add auth to dashboard        5
+GX-1198		Closed     	Dockerize dashboard repo     8
+```
+
+_Issue creation (under development)_ -- One feature in particular that was hard to find in existing tools was issue creation from template files. We've found writing stories in a text editor to be faster than filling out fields in the Jira UI. We'd like developers to be able to fill in a Jira template stored in github and run a `jiractl` command to upload it. This workflow would also ensure our stories follow a set, peer-reviewed format, without needing to copy template issues in the Jira UI. We're working on adding this feature to jiractl soon, but for now, you can retrieve and update existing issues using jiractl.
+
+To update an issue:
+```console
+$ jiractl update issue { issue_key } --points=8 --assignee=foo`
+```
+
+### What’s next?
+
+We’ve already started using this in some of our teams, but to meet our goal of sprint planning and refinement on the command line we’ll continue adding features like voting on story points and creating new sprints.
```

Comment:
This seems like a good opportunity to recommend that people use github issues for the jiractl repo to suggest new features.